Firefox/update.rdf

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<RDF:RDF xmlns:RDF="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+
+  <!-- This Description resource includes all the update and compatibility information for
+       a single add-on with the id @new-xkit. -->
+  <RDF:Description about="urn:mozilla:extension:@new-xkit">
+    <em:updates>
+      <RDF:Seq>
+
+        <!-- Each li is a different version of the same add-on -->
+        <RDF:li>
+          <RDF:Description>
+            <em:version>7.7.1</em:version> <!-- This is the version number of the add-on -->
+
+            <!-- One targetApplication for each application the add-on is compatible with -->
+            <em:targetApplication>
+              <RDF:Description>
+                <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
+                <em:minVersion>35.0</em:minVersion>
+                <em:maxVersion>44.0</em:maxVersion>
+
+                <!-- This is where this version of the add-on will be downloaded from -->
+                <em:updateLink>https://github.com/new-xkit/XKit/releases/download/v7.7.1/new-xkit-7.7.1.xpi</em:updateLink>
+
+                <!-- A page describing what is new in this updated version -->
+                <em:updateInfoURL>http://new-xkit-extension.tumblr.com/post/130919884664/new-xkit-771-release-firefox-users-must</em:updateInfoURL>
+              </RDF:Description>
+            </em:targetApplication>
+
+            <!-- Support Fennec (Firefox for Android) as well -->
+            <em:targetApplication>
+              <RDF:Description>
+                <em:id>{aa3c5121-dab2-40e2-81ca-7ea25febc110}</em:id>
+                <em:minVersion>35.0</em:minVersion>
```

Comment:
I have no reason to believe that they should be different. This is based off of the `install.rdf` generated by JPM